### ISSUE TITLE
[Accessibility]A11y_VSCode_UserSettings_ScreenReader: JAWS does not read the name of the list items properly it reads "Default settings…." for all the elements  i.e.(Commonly Used,Editor,Workbench………..etc)

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52482]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.3.1
ScreenReader Versions : JAWS18.0.5038

**Repro Steps:**
1. Launch VS Code while JAWS is ON.
2. Navigate to Settings icon in Activity Bar->Press Enter->Select "Settings" from dropdown

**Actual:**
JAWS does not read the name of the list items properly it reads "Default settings…." for all the elements  i.e.(Commonly Used,Editor,Workbench………..etc)

**Expected:**
JAWS should read the name of the list items properly   i.e.(Commonly Used,Editor,Workbench………..etc)

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The user will not be able to get clarity about control's name,state,role,value.

**MAS Reference:**
MAS 1.3.1 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment For Reference:**
[A11y_VSCode_UserSettings_JAWS2.pptx](https://github.com/Microsoft/vscode/files/2119533/A11y_VSCode_UserSettings_JAWS2.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by auchenberg** [comment]
Closing this issue as it's only observed in JAWS, and this isn't strictly needed to meet MAS C compliance.
